### DISCUSSION TITLE
help?

### DISCUSSION THREAD

**Discussion by onorosaurosrex** [KDE/ghostwriter#663] [General]
Guys, I'm new around there is a lot to learn for me,
I'm trying to use ghostwriter, but I get this:

E: Unable to locate package qtbase5-dev
E: Unable to locate package libqt5svg5-dev
E: Unable to locate package qtmultimedia5-dev
E: Unable to locate package qtwebengine5-dev
E: Package 'libqt5concurrent5' has no installation candidate
E: Unable to locate package qttools5-dev-tools

can you help?

**Comment by abenson** [comment]
I'm guessing Linux? What distribution?

**Comment by onorosaurosrex** [comment]
ubuntu

**Comment by tildagail** [comment]
You have two options.

Install the dependencies using this command:

    $ sudo apt install g++ qtbase5-dev libqt5svg5-dev qtmultimedia5-dev qtwebengine5-dev libhunspell-dev pkg-config libqt5concurrent5 qttools5-dev-tools

...and then try again to install a manually downloaded .deb file (or run your build if that's what you were doing).

Or, add the PPA to your system and install using this set of commands, which will identify and install the dependencies automatically:

    $ sudo add-apt-repository ppa:wereturtle/ppa
    $ sudo apt update
    $ sudo apt install ghostwriter

source: [https://github.com/wereturtle/ghostwriter/blob/master/README.md](https://github.com/wereturtle/ghostwriter/blob/master/README.md)

**Comment by onorosaurosrex** [comment]
Still error message

E: Unable to locate package qtbase5-dev
E: Unable to locate package libqt5svg5-dev
E: Unable to locate package qtmultimedia5-dev
E: Unable to locate package qtwebengine5-dev
E: Package 'libqt5concurrent5' has no installation candidate
E: Unable to locate package qttools5-dev-tools

Carlos Oñoro
